src/globus_cli/commands/collection/show.py

```diff
@@ -0,0 +1,87 @@
+import click
+
+from globus_cli.endpointish import Endpointish
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import collection_id_arg, command
+from globus_cli.principal_resolver import default_identity_id_resolver
+from globus_cli.services.gcs import connector_id_to_display_name, get_gcs_client
+from globus_cli.termio import FORMAT_TEXT_RECORD, formatted_print
+from globus_cli.utils import filter_fields, sorted_json_field
+
+STANDARD_FIELDS = [
```

Comment:
These are the current fields in use by GCS CLI. I'll freely admit that I didn't look too closely at them.
However, I notice that there's use of `policies.project` (last line), which our text formatter doesn't support as a way of specifying a nested field. It only passes tests because it gets filtered out by the `filter_fields` pass.

I think I'll remove that line, rather than work on supporting it today.

Reply:
I made changes to support `.` separation in the formatting, since it's pretty much trivial (when combined with `filter_fields`).
But I still removed the Google Storage-specific field from here, since it's not something we'll be supporting with `collection update` until the SDK has support for GCS Collection Policy documents.